deployment/scripts/.env

```diff
@@ -0,0 +1,16 @@
+# k8s envs
+k8s_provider="eks"
+k8s_namespace="fuel-core"
+
+# Faucet envs
+fuel_committer_image_repository="ghcr.io/fuellabs/fuel-block-committer"
```

Comment:
IMO, this should be set as default in the Chart's default `values.yaml` file. Same for image tag, but never use `latest` for image tag

Reply:
We use envsubt on the default values of the chart in our current setup, so until we migrate to kustomize this is the way we provide defaults currently.